### PR TITLE
Add ARM64 SIMD bitwise operations

### PR DESCRIPTION
**Pull request by gergocs**
Signed-off-by: Gergo Csizi gergocs@inf.u-szeged.hu

### CODE REVIEW

File: src/jit/SimdArm64Inl.h [hunk 1]
```diff
@@ -264,3 +294,25 @@ static void emitBinarySIMD(sljit_compiler* compiler, Instruction* instr)
...
+    Operand* operands = instr->operands();
+    JITArg args[3];
+
+    simdOperandToArg(compiler, operands, args[0], SLJIT_SIMD_MEM_ELEM_128, SLJIT_FR2);
+    simdOperandToArg(compiler, operands + 1, args[1], SLJIT_SIMD_MEM_ELEM_128, SLJIT_FR1);
+    simdOperandToArg(compiler, operands + 2, args[2], SLJIT_SIMD_MEM_ELEM_128, SLJIT_FR0);
+
+    simdEmitOp(compiler, SimdOp::bsl, args[2].arg, args[0].arg, args[1].arg);
+
+    JITArg jitDST;
```
Comment (zherczeg) [inline src/jit/SimdArm64Inl.h:311]: dst is enough, or just reuse args[0].

**Review by zherczeg — CHANGES_REQUESTED** [review]
Good patch

**Comment by gergocs** [comment]
patch is ready for review

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**